ui/components/ui/confusable/confusable.component.js

```diff
@@ -1,39 +1,48 @@
 import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
 import { confusables } from 'unicode-confusables';
+import { v4 as uuidv4 } from 'uuid';
 import Tooltip from '../tooltip';
 import { useI18nContext } from '../../../hooks/useI18nContext';
+import { Text } from '../../component-library';
 
-const Confusable = ({ input }) => {
+const Confusable = ({ input, asText, confusableWrapperName = '' }) => {
   const t = useI18nContext();
   const confusableData = useMemo(() => {
     return confusables(input);
   }, [input]);
 
-  return confusableData.map(({ point, similarTo }, index) => {
+  return confusableData.map(({ point, similarTo }) => {
     const zeroWidth = similarTo === '';
     if (similarTo === undefined) {
-      return point;
+      return asText ? <Text key={uuidv4()}>{point}</Text> : point;
     }
     return (
       <Tooltip
-        key={index.toString()}
+        key={uuidv4()}
```

Comment:
Why is this necessary now?

Reply:
using just the index can mean that with multiple entries multiple points can have the same key